app/src/main/java/org/mozilla/focus/telemetry/FirebaseEvent.java

```diff
@@ -34,6 +39,8 @@
     static final int MAX_PARAM_SIZE = 25;
     static final int MAX_LENGTH_PARAM_NAME = 40;
     static final int MAX_LENGTH_PARAM_VALUE = 100;
+    private static final String TAG = "FirebaseEvent";
+    private static ArrayList<String> existingPreferenceKey = new ArrayList<>();
```

Comment:
NIT: consider using HashSet for this operation. HashSet is O(1) search and O(1) insert, while ArrayList is O(n) search and O(n) insert. We don't need duplication and we don't need get for this data.